docs/en/pipeline-mgmt/data-processing/configure-execution-para.md

```diff
@@ -22,11 +22,9 @@ Syntax
 
 ## Config Spark job parameters
 
-By default, the EMR Serverless job is configured with [default settings][jobs-spark], which is suitable for most cases, e.g., hourly processing.
+By default, Clickstream evaluates the processing data set intelligently and set EMR job parameters appropriately. In most of time, you do not need to adjust the EMR job parameters, but if you want to override the EMR job parameters, you can put `spark-config.json` file in S3 bucket to set your job parameters.
```

Comment:
By default, the Clickstream pipeline automatically adjusts EMR job parameters based on the dataset volume that requires processing.

Reply:
changed.